tests/e2e/deployment-test.sh

```diff
@@ -96,7 +96,7 @@ writerIdentity=$(gcloud logging sinks describe "${LOG_ROUTER}" --format json | j
 gcloud pubsub topics add-iam-policy-binding "${PUBSUB_TOPIC}" --member ${writerIdentity} --role roles/pubsub.publisher > /dev/null 2>&1
 
 # Create E2E Sample App
-gcloud functions deploy sample_app \
+gcloud functions deploy "${CLOUD_FUNCTION_NAME}" \
 --runtime python37 \
```

Comment:
This is out of the scope of this task, but please consider changing the test runtime to python38 as this is the version that we are using.

Reply:
It's just a sample app which produces logs for e2e tests, it's not related with our processing logs function

Reply:
you must change also here https://github.com/dynatrace-oss/dynatrace-gcp-function/pull/139/files#diff-ae94410f78e789a88f96202c7f4a8ef10ef33ec028de15ecf6ecc80b4c9bcfadR162 because it will fail